Cargo.toml

```diff
@@ -30,7 +30,6 @@ with_ctap1 = ["crypto/with_ctap1"]
 with_nfc = ["libtock_drivers/with_nfc"]
 
 [dev-dependencies]
-elf2tab = "0.6.0"
 enum-iterator = "0.6.0"
```

Comment:
And do we use this one? :)

Reply:
It's part of some tests. Basically doing something like
for variant in enum
  convert from and to CBOR
  assert_eq!